src/main_nep/nep4.cu

```diff
@@ -261,18 +264,27 @@ static __global__ void apply_gnn_message_passing(
     // get messages for atom i and neighbors
     float q_theta_i[MAX_DIM] = {0.0f};
     float q_theta_j[MAX_NEIGHBORS * MAX_DIM] = {0.0f}; // maximum size when all atoms are neighbors
+    float fc_ij[MAX_DIM] = {0.0f};
```

Comment:
MAX_DIM should be MAX_NEIGHBORS

Reply:
Good catch, I mixed them up in my head :rofl: